docs/dev-guides/howto-use-applicationinsights-metrics.md

```diff
@@ -0,0 +1,117 @@
+# How to use Application Insights metrics
+
+[Home](readme.md)
+
+All the cloud microservices of Industrial IoT platform have been instrumented with Application Insights already.  The instrumentation monitors your app and sends telemetry data to Application Insights. 
```

Comment:
Please remove double space: `already.  The`

Reply:
I think so too.